lib/datasets/pmjt-dataset-list.rb

```diff
@@ -0,0 +1,68 @@
+module Datasets
+  class PMJTDatasetList < Dataset
+    LATEST_VERSION = '201901'
+    URL_FORMAT = "http://codh.rois.ac.jp/pmjt/list/pmjt-dataset-list-%{version}.csv".freeze
+
+    Record = Struct.new(:unit,
+                        :open_data_category,
+                        :tag,
+                        :release_time,
+                        :number,
+                        :pubished_or_copied,
+                        :publication_year,
+                        :original_number,
+                        :id,
+                        :title,
+                        :text,
+                        :bibliographical_introduction,
+                        :year)
+
+    def initialize
+      super()
+      @metadata.id = "pmjt-dataset-list-#{LATEST_VERSION}"
+      @metadata.name = "List of pre-modern Japanese text dataset"
+      @metadata.url = URL_FORMAT % {version: LATEST_VERSION}
+      @metadata.licenses = ["CC-BY-SA-4.0"]
+      @metadata.description = <<~DESCRIPTION
+        Pre-Modern Japanese Text, owned by National Institute of Japanese Literature, is released image and text data as open data.
+        In addition, some text has description, transcription, and tagging data.
+      DESCRIPTION
+
+      @data_path = cache_dir_path + (@metadata.id + ".csv")
+    end
+
+    def each(&block)
+      return to_enum(__method__) unless block_given?
+
+      download(@data_path, @metadata.url) unless @data_path.exist?
+      CSV.open(@data_path, headers: :first_row, encoding: "Windows-31J:UTF-8") do |csv|
+        csv.each do |row|
+          record = prepare_record(row)
```

Comment:
How about `create_record(row)`, `build_record(row)` or something?
I think that `prepare` is generally used for preparing an existing object. But this use case a record is created in `prepare_record` instead of creating a record in this method and passing the created record to `prepare_record`. So I think that `create_`, `build_` or something is better.